File: learn_c++/25-06-25/simple_interest.cpp
#include <iostream>
using namespace std;

int main()
{
    const double rate = 0.075;
    double principal;
    int time;

    cout << "Enter principal amount: ₹";
    cin >> principal;

    cout << "Enter time in years: ";
    cin >> time;

    double interest = principal * rate * time;
    cout << "Simple Interest = ₹" << interest << endl;

    cout << "Total amount = ₹" <<  principal + interest << endl;

    return 0;
}

File: learn_python/09-08-25/simple_interest.py
try:
    principal = float(input("Principal: "))
    rate_of_interest = float(input("Interest rate: "))
    period = int(input("Time period (years): "))
    SI = (principal * rate_of_interest * period) / 100
    print(f"Simple Interest: {SI}")
except ValueError:
    print("Please enter valid numeric input!")

